packages/icon/src/react/__specs__/__snapshots__/storyshots.spec.js.snap

```diff
@@ -2,1891 +2,1514 @@
 
```

Comment:
these snapshots should be really close. i'm mocking the storybook theme decorator so you'll see that an extra div is removed